client/src/style/feedback.css

```diff
@@ -0,0 +1,75 @@
+.containerFeedback{
+    border-radius: 10px;
+    border: 1px solid #ccc;
+    padding: 10px;
+    margin: 10px;
+    box-shadow: 0 0 5px rgba(0,0,0,0.5);
+    width: auto; 
+}
+
+.btnFormSend{
+    margin-top: 10px;
+}
+
+.showMoreComm{
+    width: 100%; 
+    height: 15px; 
+    border-bottom: 1px solid white; 
+    text-align: center;
+    margin-left: auto;
+}
+.showMoreCommSpan{
+    font-size: 15px;
+    padding: 0 10px;
+    background-color: #1C1B1E;
+}
+.showMoreCommSpan:hover{
+    color: blue;
+    cursor: pointer;
+}
+
+.textAreaFeedback:focus{
+    box-shadow: 0 0 0 1px rgba(41, 176, 217, .25);
+}
+
+.feedbackUserContainer{
+    display: inline-flex;
+    width: 100%;
+    margin-top: 15px;
+}
+.feedbackMess{
+    width: 100%;
+    margin: 0px 10px;
+}
+
+.hrStyle{
+    padding: 0;
+    margin: 0;
+    background-color: white;
+}
+.nameDate{
+    margin-right: 35px;
+    font-size: 12px;
+}
+
+.answerToFeedback{
+    margin: 0px 30px 0px 10px;
+    padding: auto;
+}
+.answerToFeedback:hover{
+    color: blue;
+    cursor: pointer;
+}
+
+.complaintFeddback{
+    float: right;
+    margin-right: 10px;
+}
+.complaintFeddback:hover{
+    cursor: pointer;
+}
+
+
+.likeDislike:hover{
+    cursor: pointer;
+}
```

Comment:
Great! Just create another files to store separate `.js` files styles, not all together.

Reply:
we not need .css file for each one component